### PR TITLE
 docs: add docs for programmatic logout

### LINKED ISSUE #17016: [Security] Add a method in the security helper to ease programmatic log…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#41406
| PR author(s) | @johnkrovitch, @chalasr
| Merged in    | 6.2

### PR DESCRIPTION
**Pull request by johnkrovitch**
This PR adds documentation for the new feature "programmatic logout" merge in the PR https://github.com/symfony/symfony/pull/41406

### CODE REVIEW

File: security.rst [hunk 1]
```diff
@@ -1723,6 +1723,49 @@ Next, you need to create a route for this URL (but not a controller):
...
+
+    use App\Security\Authenticator\ExampleAuthenticator;
+    use Symfony\Bundle\SecurityBundle\Security\Security;
+
+    class SecurityController
+    {
+        public function someAction(Security $security): Response
+        {
+            // logout the user in on the current firewall
+            $response = $this->security->logout();
```
Comment (94noni) [inline security.rst]:
```suggestion
            $response = $security->logout();
```

or inject in constructor :)

File: security.rst [hunk 2]
```diff
@@ -1723,6 +1723,49 @@ Next, you need to create a route for this URL (but not a controller):
...
+    method was introduced in Symfony 6.2.
+
+You can logout user programmatically using the `logout()` method of the
+:class:`Symfony\\Bundle\\SecurityBundle\\Security\\Security` helper. The user will be logout from the current firewall
+in the request. If the current request is not behind a firewall a ``\LogicException`` will be thrown. ::
+
+    // src/Controller/SecurityController.php
+    namespace App\Controller\SecurityController;
+
+    use App\Security\Authenticator\ExampleAuthenticator;
```
Comment (94noni) [inline security.rst]: seems useless isn't it?

**Comment by wouterj** [comment]
Thank you @johnkrovitch!

**Comment by johnkrovitch** [comment]
Thank you !

**Pull request merged**